src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+                <p>WhatsApp, bel of mail ons voor verdere vragen.
+                </p>]]>
+            <ad:description/>
+======= =========================================================================================================
+
+
+.. index:: categoryId
+.. _feed_categoryId:
+
+categoryId
+""""""""""
+
+The category in which the ad is placed in the :ref:`categories` tree.
+
+Each ad belongs to one and only one category and the category of an ad cannot be changed.
+This field can only be set once during the lifecycle of an ad.
+
+An integer value from the category list. Must be an id of a category with a
+non-zero parent id.
+
+======= ====================================================
+Example	.. code-block:: html
+        
+         <ad:categoryId>945</ad:categoryId>
+======= ====================================================
+
+.. index:: status
+.. _feed_status:
+
+status
+""""""
+
+Ad status. One of the following:
+
+====== ====================================================
+Name   Description
+====== ====================================================
+ACTIVE The ad is active, it can be found on the website
+PAUSED The ad is paused, it cannot be found on the website
+====== ====================================================
+
+The provided (desired) status may differ from the resulting one, depending on the other conditions.
+For instance, budget may be depleted, or the seller may have too many active ads already in the category.
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:status>PAUSED</ad:status>
+======= =============================
+
+.. index:: url
+.. _feed_url:
+
+url
+"""
+
+An external URL that is shown on the ad page. This must be a valid http(s)
+url.
+
+======= =======================================
+Example .. code-block:: html
+        
+         <ad:url>https://www.bmw.de</ad:url>
+======= =======================================
+
+.. index:: vanityUrl
+.. _feed_vanityUrl:
+
+vanityUrl
+"""""""""
+
+The text/url that will be displayed instead of the url in :ref:`feed_url`.
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:vanityUrl>BMW</ad:vanityUrl>
+======= ==================================
+
+.. index:: priceType
+.. _feed_priceType:
+
+priceType
+"""""""""
+
+Must be a valid price type identifier from the list of :ref:`price_types`.
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:priceType>FIXED_PRICE</ad:priceType>
+======= ============================================
+
+.. index:: price
+.. _feed_price:
+
+price
+"""""
+
+The meaning of the value depends on the :ref:`feed_priceType`. 
+
+If it is `FIXED_PRICE` or `BIDDING_FROM` then **price** is mandatory and needs to be greater than 0.
+The maximum allowed **price** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
```

Comment:
I think it is cents, please double check.

Reply:
It's inconsistent with our XSD at least 

```
                price                       | ad price in cents if applicable                | Yes for FIXED_PRICE and BIDDING_FROM, No otherwise
```

Reply:
Yes, this should be in cents. We do not do full euros (with floats) anywhere.

Reply:
fixed